test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3803,13 +3803,19 @@ def forward(self, input, other):
         y = torch.randn(1, 2, 1)
         self.run_test(RemainderModel(), (x, y))
 
+    # Note: Integral Tensor remainder to floating-point scalar (not Tensor)
```

Comment:
this is bc-breaking change. Can we still make this case working for the original test? 

Otherwise, existing pipeline might just break. cc: @spandantiwari, @BowenBao

Reply:
@ejguan could you try removing the casting at here

https://github.com/pytorch/pytorch/blob/dee82ef3ea1ed2cae4e891f071adff03cb4a6e0d/torch/onnx/symbolic_opset9.py#L2598

and see if it works for the original test case? The was workaround of onnx export for remainder since it was not adhering to type promotion before.